tests/unit/test_output.py

```diff
@@ -85,3 +89,21 @@ def test_merge_history():
     output.merge_history(root, file_type, source1, source2)
     assert utils.is_timestamp(f"-{root.history[:19]}") is True
     assert root.history[19:] == ' - dummy file created\nsome history x\nsome history y'
+
+
+def test_get_old_uuid(fake_nc_file):
+    assert output._get_old_uuid(True, fake_nc_file) == 'abcde'
+    assert output._get_old_uuid(False, fake_nc_file) is None
+    assert output._get_old_uuid(True, 'new_file.nc') is None
+    assert output._get_old_uuid(False, 'new_file.nc') is None
+
+
+def test_copy_variables(tmpdir_factory, fake_nc_file):
+    file = tmpdir_factory.mktemp("data").join("nc_file.nc")
+    root_grp = netCDF4.Dataset(file, "w", format="NETCDF4_CLASSIC")
+    output._add_standard_global_attributes(root_grp, 'abcd')
+    assert root_grp.file_uuid == 'abcd'
```

Comment:
I thought that you can have only one assertion in one test concluding pytest.mark.parametrize. So if first one breaks, other are not checked. Maybe in this test it doesn't matter.

Reply:
This function looks confusing because its name is incorrect. Correct name would be `test_that_uuid_changes_in_rewriting` or something like that. The the two assertions in the same test function make more sense. I will fix this.